test/modules/EdgeHubRestartTester/src/DirectMethodEdgeHubConnectorTest.cs

```diff
@@ -79,7 +79,7 @@ class DirectMethodEdgeHubConnectorTest : IEdgeHubConnectorTest
                     MethodRequest request = new MethodRequest(
                         directMethodName,
                         Encoding.UTF8.GetBytes($"{{ \"Message\": \"Hello\", \"DirectMethodCount\": \"{this.directMethodCount}\" }}"),
-                        Settings.Current.SdkOperationTimeout,
+                        TimeSpan.FromSeconds(5),   // Minimum value accepted by SDK
```

Comment:
how did you know this? https://github.com/Azure/azure-iot-sdk-csharp/blob/58d2379f8a556e67098f7283223c328822819d9e/iothub/device/src/MethodInvokeRequest.cs#L64; it should accept any number of seconds.

Reply:
I gives me an error saying the number must be between 5 to 300 seconds.